### DISCUSSION TITLE
How are the databases updated

### DISCUSSION THREAD

**Discussion by getorca** [JerBouma/FinanceDatabase#89] [General]
how are the databases are updated? I don't see any scripts for updating the databases in the current repo.

**Comment by JerBouma** [comment]
GitHub Actions updates the Equities database every Sunday with new companies that performed IPOs and changing market capitalizations. Other databases are stationary and rely on community involvement.

**Comment by getorca** [comment]
Thanks, I missed the github actions. Looks like it's only for the 3 major US exchanges, the others countries/exchanges are community updates as well?

**Reply by JerBouma** [reply]
This is correct, the Finance Database is a great _initial_ database to use for data analysis but when you decide to actively trade or commercialise a model based on the database, you should look elsewhere (= paid subscriptions such as Bloomberg Terminal).
